### DISCUSSION TITLE
Sinope TH1123ZB power stays to NULL

### DISCUSSION THREAD

**Discussion by rigorm** [Koenkk/zigbee2mqtt#10645] [Devices]
Hi,
I have a couple of Sinope TH1123ZB line thermostat and all of them have the power (watts) value to null. they are fairly new (got them at the end of november). I do have voltage and current available though. In the reporting tab, there is no cluster "haElectricalMesurement" with attribute "activepower".  If I try to set it, I get the error: 

`Request 'zigbee2mqtt/bridge/request/device/configure_reporting' failed with error: 'ConfigureReporting 0x500b914000029683/1 haElectricalMeasurement([{"attribute":"activePower","minimumReportInterval":60,"maximumReportInterval":3600,"reportableChange":0}], {"timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":true,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null,"writeUndiv":false}) failed (Status 'UNREPORTABLE_ATTRIBUTE')'`

If I go to the dev console and set the cluster and attribute, I does return the power value.

zigbee2mqtt says that the thermostat's firmware version and release date are : 

`Firmware build date: 20200330, Firmware version: 2051`

Zigbee stick is a Sonoff Zigbee 3.0 Plus with zStack 

`Coordinator type zStack3x0, Coordinator revision 20210708`

i'm using the latest version as of this writing (zigbee2mqtt 1.22.2.-1)

Anyone can chime on my problem ?

Thanks,

Jeff C.

**Comment by lucasec** [comment]
Unrelated to this project, but I am seeing this behavior too trying to implement this device into OpenHAB. If I configure reporting with a short maximum reporting interval, it sends me regular reports of these attributes. But it refuses to send reports when any "reportable change" threshold is crossed. All of the other normal attributes on the device report fine.

Best I can tell, it may be a firmware bug. What is odd is I also have the TH1500ZB and it reports the electrical measurement attributes just fine (honoring the "reportable change" parameter).
